responses/nl/HassGetState.yaml

```
@@ -1,4 +1,55 @@
 language: nl
 responses:
   intents:
-    HassGetState: {}
+    HassGetState:
+      one: "{{ slots.name | capitalize }} is {{ state.state_with_unit }}"
+      one_yesno: |
+        {% set translations =
+          {
+            "on": "aan",
+            "off": "uit",
+            "open": "open",
+            "closed": "dicht",
+            "opening": "aan het openen",
+            "closing": "aan het sluiten"
+          }
+        %}
+        {% if state.state_with_unit in translations %}
+          {% set translated = translations[state.state_with_unit] %}
+        {% endif %}
+        {% if query.matched: %}
+          Ja
+        {% else: %}
+          Nee, die is {{ translated if translated is defined else state.state_with_unit }}
+        {% endif %}
+      any: |
+        {% if query.matched: %}
+        Ja, {{ query.matched | map(attribute="name") | sort | join(", ") }}
```

let's indent text between if statements.
```suggestion
          Ja, {{ query.matched | map(attribute="name") | sort | join(", ") }}
```